code/datums/outfits/jobs/command.dm

```diff
@@ -0,0 +1,41 @@
+/decl/hierarchy/outfit/job/captain
+	name = OUTFIT_JOB_NAME("Captain")
+	head = /obj/item/clothing/head/caphat
+	glasses = /obj/item/clothing/glasses/sunglasses
+	uniform = /obj/item/clothing/under/rank/captain
+	l_ear = /obj/item/device/radio/headset/heads/captain
+	shoes = /obj/item/clothing/shoes/color/brown
+	id_type = /obj/item/weapon/card/id/gold
+	pda_type = /obj/item/modular_computer/pda/captain
+	backpack_contents = list(/obj/item/weapon/storage/box/ids = 1)
+
+/decl/hierarchy/outfit/job/captain/New()
+	..()
+	backpack_overrides[/decl/backpack_outfit/backpack]      = /obj/item/weapon/storage/backpack/captain
+	/*backpack_overrides[/decl/backpack_outfit/satchel]       = /obj/item/weapon/storage/backpack/satchel/cap
```

Comment:
Why captain satchel was removed from setup?

Reply:
I mean `/obj/item/weapon/storage/backpack/satchel/cap` exists. But it's type is `/obj/item/weapon/storage/backpack/satchel_cap`